inscontext/context.go

```diff
@@ -139,3 +140,14 @@ func WithLog(parent context.Context, clog core.Logger) *Ctx {
 func (ctx *Ctx) Log() core.Logger {
 	return ctx.log.WithField("traceid", ctx.TraceID())
 }
+
+// WithRandomTraceID returns *Ctx with random traceID (uuid format)
+func WithRandomTraceID() *Ctx {
+	traceID := ""
+	qid, err := uuid.NewV4()
+	if err == nil {
```

Comment:
please set on error some non empty value, like "gettraceidfailed", or smth like that